Makefile.helper.mk

```diff
@@ -15,6 +15,7 @@ KUBECONFIG       ?= ${HOME}/.kube/config
 export PATH := go/bin:$(PATH)
 
 patch:
+	[ -z $(rpm -qa patch) ] && dnf install patch
```

Comment:
Shall we rather add this to the CI? I have the same problem with `xz`.

Reply:
This is RHEL specific, for upstream Kube that would mostly fail.

Reply:
Haha we don't have `rpm` on the CI either.
Changing the implementation of the PR.